### PR TITLE
Bump ordered consumer flow control window back up.

### PR DESCRIPTION
**Pull request by derekcollison**
Have seen some small dips in some global systems that @ripienaar has been running and I have been testing. Originally bumped it down due to some instability but that was not associated with this in the end.

Signed-off-by: Derek Collison <derek@nats.io>

/cc @nats-io/core

### CODE REVIEW

**Review by ripienaar — APPROVED** [review]
LGTM, though i think my drop in traffic was because majority of stream leaders were on one machine, after a bit of poking to spread leaders out traffic is as expected

File: server/consumer.go [hunk 1]
```diff
@@ -295,7 +295,7 @@ const (
 	JsDeleteWaitTimeDefault = 5 * time.Second
 	// JsFlowControlMaxPending specifies default pending bytes during flow control that can be
 	// outstanding.
-	JsFlowControlMaxPending = 16 * 1024 * 1024
+	JsFlowControlMaxPending = 32 * 1024 * 1024
```
Comment (kozlovic) [inline server/consumer.go:298]: * Before was:
1MB and maxpb = limit / 16
* 19 days ago:
changed to 32MB *AND* changed maxpb = limit / 4
* 4 days ago:
changed to 16MB *AND* changed maxpb = limit / 16
* today:
changed back to 32MB but maxpb not changed... hard to follow..

**Review by kozlovic — CHANGES_REQUESTED** [review]
Check if maxpb needs to be changed too.

File: server/consumer.go (hunk 1 above)
Comment (derekcollison) [reply server/consumer.go:298]: Nope that is ok and will leave as is. Thanks.

Comment (kozlovic) [reply server/consumer.go:298]: Sorry, so are you saying that there is no need to change maxpb even if you update JSFlowConstrolMaxPending? If so, let me know and I will approve the PR.

Comment (derekcollison) [reply server/consumer.go:298]: Correct, I want it to start at /16 or 2mb

**Review by kozlovic — APPROVED** [review]
LGTM
